lib/taskana-core/src/test/java/acceptance/task/CreateTaskAccTest.java

```diff
@@ -632,32 +650,6 @@ void testThrowsExceptionIfAttachmentIsInvalid() throws ClassificationNotFoundExc
             "2018-01-15",
             createSimpleCustomProperties(3)));
 
-    testCreateTask.accept(
-        createAttachment(
-            "DOCTYPE_DEFAULT",
-            createObjectReference(
-                "COMPANY_A",
-                "SYSTEM_B",
-                null,
-                "ArchiveId",
-                "12345678901234567890123456789012345678901234567890"),
-            "E-MAIL",
-            "2018-01-15",
-            createSimpleCustomProperties(3)));
-
-    testCreateTask.accept(
-        createAttachment(
-            "DOCTYPE_DEFAULT",
-            createObjectReference(
-                "COMPANY_A",
-                null,
-                "INSTANCE_B",
-                "ArchiveId",
-                "12345678901234567890123456789012345678901234567890"),
-            "E-MAIL",
-            "2018-01-15",
-            createSimpleCustomProperties(3)));
-
```

Comment:
Why did you remove this?

Reply:
Because the test expected an exception which is not thrown any more when ObjecReference#System/System_Instance is not set.